test/test_beam_manager.py

```diff
@@ -0,0 +1,264 @@
+import json
+
+from biggerquery.beam_manager import create_dataflow_manager, DataflowManager, TemplatedBeamManager, BeamManager
+from test.beam_test_case import BeamTestCase
+import apache_beam as beam
+from apache_beam.options.pipeline_options import \
+    PipelineOptions, StandardOptions
+from mock import patch
+
+EXAMPLE_SAMPLE_SCHEMA = '''
+{"namespace": "example.avro",
+ "type": "record",
+ "name": "exampleSample",
+ "fields": [
+     {"name": "id", "type": "string", "mode": "required"},
+     {"name": "name", "type": "string", "mode": "required"}
+ ]
+}
+'''
+
+EXAMPLE_OUTPUT_SCHEMA = '''
+{"namespace": "example.avro",
+ "type": "record",
+ "name": "exampleSample",
+ "fields": [
+     {"name": "id", "type": "string", "mode": "required"},
+     {"name": "name", "type": "string", "mode": "required"}
+ ]
+}
+'''
+
+
+class Data:
+    def __init__(self, data):
+        self.data = data
+
+    def contains(self, expected_data):
+        matching_element = [
+            s for s in self.data
+            if expected_data['id'] == s['id'] and expected_data['name'] == s['name']
+        ]
+        return len(matching_element) == 1
+
+    def __len__(self):
+        return len(self.data)
+
+
+class DataFlowManagerTest(BeamTestCase):
+    def setUp(self):
+        super(DataFlowManagerTest, self).setUp()
+        self.data_flow_manager = create_dataflow_manager(
+            project_id='project',
+            runtime="2019-01-01",
+            dataset_name='test_dataset',
+            dataflow_bucket="bucket",
+            internal_tables={"internal_table": "project.dataset.internal_table"},
```

Comment:
internal_tables should be a list of table names(manager can construct a full table id from given project_id and dataset_name.

Reply:
Look now please.